matchzoo/pack.py

```diff
@@ -5,39 +5,46 @@
 from matchzoo import datapack
 
 
-def segment(inputs: list, stage: str) -> datapack.DataPack:
+def has_label(data: list):
+    if data and len(data[0]) == 5:
+        return True
+    elif len(data[0]) == 4:
+        return False
+    else:
+        raise ValueError('Invalid data format.')
```

Comment:
remove this function since we already have a property defined in datapack.py?

Reply:
This is necessary.